automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookDataSyncingSpec.scala

```diff
@@ -59,13 +61,16 @@ class NotebookDataSyncingSpec extends ClusterFixtureSpec with NotebookTestUtils
                 remoteContentSize shouldBe localContentSize
               }
 
+              val hashedUser = Some(String.format("%064x", new BigInteger(1, MessageDigest.getInstance("SHA-256").digest((gcsPath.bucketName + ":" + clusterFixture.cluster.creator).getBytes("UTF-8")))))
```

Comment:
I'd like to see this split into two lines with the `MessageDigest....` as its own val with a meaningful name. I find it a bit hard to read as is.